src/remark/models/slideshow.js

```diff
@@ -63,6 +63,9 @@ function Slideshow (events, dom, options, callback) {
   if (options.sourceUrl) {
     loadFromUrl(options.sourceUrl, callback);
```

Comment:
Refactor `loadFromUrls` and use it here -- (that way we can remove the `loadFromUrl` function and only have one.